upgrade/breaking-changes/4-0-0 .md

```diff
@@ -0,0 +1,308 @@
+</table>
+
+### Form
+
+- The Form columns can have different widths
+
+### Grid
+
+- Removed default filter descriptors in the state when the Grid `FilterMode` is set to `FilterMenu`.
+- Removed `Primary` parameter from the `<GridCommandButton>`.
+- Removed `NumberFormat` and `Width` from `ExcelExportableColumn` for [CSV export]({%slug grid-export-events%}#for-csv-export).
+
+### MaskedTextBox
+
+- Changed the signature of the `PlaceHolder` parameter to `Placeholder`.
+
+>caption Placeholder parameter in UI for Blazor up to version 3.8.0 and after version 4.0.0
+
+<table>
+    <thead><tr>
+        <th>UI for Blazor 3.8.0</th>
+        <th>UI for Blazor 4.0.0</th>
+    </tr></thead>
+    <tbody>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TelerikMaskedTextBox PlaceHolder="The placeholder text" /&gt;
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TelerikMaskedTextBox Placeholder="The placeholder text" /&gt;
+</code></pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### SplitButton
+
+- Removed the `Shape` parameter.
+
+### TextArea
+
+- Changed the signature of the `PlaceHolder` parameter to `Placeholder`.
+
+>caption Placeholder parameter in UI for Blazor up to version 3.8.0 and after version 4.0.0
+
+<table>
+    <thead><tr>
+        <th>UI for Blazor 3.8.0</th>
+        <th>UI for Blazor 4.0.0</th>
+    </tr></thead>
+    <tbody>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TelerikTextArea PlaceHolder="The placeholder text" /&gt;
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TelerikTextArea Placeholder="The placeholder text" /&gt;
+</code></pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### TextBox
+
+- Changed the signature of the `PlaceHolder` parameter to `Placeholder`.
+
+>caption Placeholder parameter in UI for Blazor up to version 3.8.0 and after version 4.0.0
+
+<table>
+    <thead><tr>
+        <th>UI for Blazor 3.8.0</th>
+        <th>UI for Blazor 4.0.0</th>
+    </tr></thead>
+    <tbody>
+        <tr>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TelerikTextBox PlaceHolder="The placeholder text" /&gt;
+</code></pre>
+            </td>
+            <td style="vertical-align:top">
+<pre><code>
+&lt;TelerikTextBox Placeholder="The placeholder text" /&gt;
+</code></pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### ToggleButton
+
+- Removed the `Shape` parameter.
+
+### Toolbar
+
+- Changed the default value of the `Adaptive` parameter from `false` to `true`.
+
+### TreeList
+
+- Removed default filter descriptors in the state when the TreeList `FilterMode` is set to `FilterMenu`.
+- Removed `Primary` parameter from the `<TreeListCommandButton>`.
+
+>important Any snippet created before `Telerik.UI.for.Blazor` version 4.0 (January 18, 2023) may need an update due to the above list of changes.
```

Comment:
Some customers may find this sentence ambiguous. Are we talking specifically about REPL as in the 3.0 article? For regular production apps this should be common sense?